fluentgen/src/main/java/com/azure/autorest/fluent/FluentGen.java

```diff
@@ -167,6 +168,7 @@ protected NodeTuple representJavaBeanProperty(Object javaBean, Property property
 
     Client handleMap(CodeModel codeModel) {
         JavaSettings settings = JavaSettings.getInstance();
+        FluentStatic.setFluentJavaSettings(getFluentJavaSettings());

```

Comment:
where is FluentJavaSetting used?

Reply:
It's used here for determine the operation group name for operations without group: 
https://github.com/XiaofeiCao/autorest.java/blob/scenario_test_gen/fluentgen/src/main/java/com/azure/autorest/fluent/mapper/FluentProxyMethodMapper.java#L78